src/main/views/lists/tabs/contactTypes/editContactType.njk

```diff
@@ -0,0 +1,71 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set isNewContactType = true %}
+{% if contactType.id %}
+  {% set isNewContactType = false %}
+{% endif %}
+
+{% if isNewContactType %}
```

Comment:
Can maybe shorten to: 

```
{% if !contactType.id %}
  <h2 class="govuk-heading-l">Add New Contact Type</h2>
{% else %}
  <h2 class="govuk-heading-l">Editing Contact Type: {{contactType.type}}</h2>
{% endif %}
```

And then maybe we can remove lines 5 to 8?

Reply:
refactored with your suggestion